ConnectedCanteens/ngsild-payloads/BatchCreateEntities.json

```diff
@@ -76,8 +76,8 @@
             "object": "urn:ngsi-ld:DeviceModel:01"
         },
         "controlledAsset": {
-            "type": "Relationship",
-            "object": "urn:ngsi-ld:Trash:bread"
+            "type": "Property",
```

Comment:
the controlled asset is expected to be the entity in which a property is updated (so, in this case, the restaurant)